.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# These developers will be requested for review when someone opens a pull request.
+# Uncomment the line below and mention the reviewers with their github handles
+# *       @webchapter
```

Comment:
@noor-ahmed Not sure when are we going to use this 🤔  because we mostly request the review from developers who familiar with the context of the template

Reply:
IMO, We can keep it, I saw [android template](https://github.com/nimblehq/android-templates/blob/kotlin/.github/CODEOWNERS) recently added it. We can just set the default reviews of a project